src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2107,6 +2107,20 @@ For more information, see https://github.com/ow2-proactive/scheduling/blob/maste
 === Catalog
 include::./Catalog.adoc[]
 
+[[_backup]]
+== Configure automated backup
+
+To use the automated backup, edit the file `PROACTIVE_HOME/config/shared/settings.ini`.
+Enable backup mechanism by setting `pa.server.backup` to true.
+Backup will be performed according to provided cron expression in `pa.server.backup.period`.
+Set file/folders you want to backup into `pa.server.backup.targets`.
+Backup will appear in the folder specified by `pa.server.backup.destination`.
+
+Backup mechanism requires ProActive Scheduler freeze during backup.
```

Comment:
This is not clear: the backup mechanism automatically freezes the scheduler while performing the backup to guarantee data integrity.

Reply:
ok